module_utils/network_lsr/argument_validator.py

```diff
@@ -2216,6 +2217,15 @@ def _ipv6_is_not_configured(connection):
                     idx,
                     "ip.ipv6_disabled is not supported by initscripts.",
                 )
+
+            if not self.TOX_UNIT_TEST and not hasattr(
+                Util.NM(), "SETTING_IP6_CONFIG_METHOD_DISABLED"
+            ):
+                raise ValidationError.from_connection(
+                    idx,
+                    "ip.ipv6_disabled is not supported by the running version of "
+                    "NetworkManger.",
```

Comment:
Please consider enhancing the message to mention what users can do instead, IIRC there is a way to make NM ignore Ipv6 in this case.

Reply:
How about: 
```python
                raise ValidationError.from_connection(
                    idx,
                    "ip.ipv6_disabled is not supported by the running version of "
                    "NetworkManger. However, if not explicitly enabling "
                    "ip.ipv6_disabled, ip.auto6, or specifying any IPv6 address, the "
                    "role will instruct NetworkManager not to configure IPv6 for this "
                    "connection",
                )
```